src/pages/wallet-overview/wallet-overview.ts

```diff
@@ -116,18 +129,116 @@ export class WalletOverviewPage {
     this.navCtrl.push(TransferPage);
   }
 
-  /**
-   * Open the backup wallet page
-   */
-  backupWallet(): void {
-    this.navCtrl.push(RestoreBackupPage);
-  }
-
   refreshWalletBalance(): void {
     this.getWalletBalance(this.currentWallet.publicKey);
     this.getTransactionHistory(this.currentWallet.publicKey);
   }
 
+  exportPrivatekey(): boolean {
+    console.log("Export privatekey!");
+    return false;
+  }
+
+  exportKeystore(): boolean {
+    if (this.currentWallet.type !== "local") {
+      return false;
+    }
+    let alert = this.alertCtrl.create();
+    alert.setTitle('Export keystore');
+    alert.addInput({
+      type: 'radio',
+      label: 'Download file',
+      value: 'file',
+      checked: true
+    });
+    alert.addInput({
+      type: 'radio',
+      label: 'Copy to clipboard',
+      value: 'clipboard',
+      checked: false
+    });
+    alert.addButton('Cancel');
+    alert.addButton({
+      text: 'OK',
+      handler: data => {
+        let keystoreData = JSON.stringify((this.currentWallet as ILocalWallet).keyStore);
+        if (data === "clipboard") {
+          if (this.platform.is("android") || this.platform.is("ios")) {
+            this.clipboard.copy(keystoreData);
+          } else {
+            this.copyToClipboardWeb(keystoreData);
+          }
+          this.showBottomToastMessage('Copied keystore to clipboard!', 2000, "bottom");
```

Comment:
This does not appear to be multi language

Reply:
Multi language added next request